Resources/Prototypes/_DV/Entities/Mobs/Species/kitsune.yml

```diff
@@ -0,0 +1,63 @@
+- type: entity
+  save: false
+  id: BaseMobKitsune
+  name: Urist McKitsune
+  abstract: true
+  components:
+  - type: Physics
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape: !type:PhysShapeCircle
+          radius: 0.35
+        density: 50
+        restitution: 0.0
+        mask:
+        - MobMask
+        layer:
+        - MobLayer
+  - type: Body
+    prototype: Kitsune
+  - type: Damageable
+    damageContainer: BiologicalMetaphysical # foxes take holy damage but at a reduced amount
```

Comment:
This makes it so that topicals don't work, since they're only set up to work on `Biological` damage containeres